packages/client/graphql_schema.json

```diff
@@ -5080,6 +5080,18 @@
               "isDeprecated": false,
               "deprecationReason": null
             },
+            {
+              "name": "totalVotes",
```

Comment:
Can we rename this to "totalVotesReceived", instead of "totalVotes", to make sure that it's more evident what this refers to? Later we may want an attribute for "totalVotesGiven", for reasons mentioned earlier in the story comments.

Reply:
Sure, thanks.